flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointCoordinator.java

```diff
@@ -857,7 +859,7 @@ public void run() {
 		// the 'min delay between checkpoints'
 		lastCheckpointCompletionNanos = System.nanoTime();
 
-		LOG.info("Completed checkpoint {} ({} bytes in {} ms).", checkpointId,
+		LOG.info("Completed checkpoint {} for job {} ({} bytes in {} ms).", checkpointId, job,
```

Comment:
maybe move job to the end of the message, i.e. `Completed checkpoint {} ({} bytes in {} ms) for job {} .`

Reply:
I actually prefer if before the brackets - it makes a bit more sense there

Reply:
Hehe, i was going to say that it makes more sense to group checkpoint information (id and size) together ;)

Won't block the PR on this though.

Reply:
I would say, that "checkpoint x for job y" has these properties, not "checkpoint x" alone